Runtime client issue when trying to run the server

Hey,

I was able to follow your README example but encountered `readonly headers: Pick<Headers, "get">;` when trying to run `yarn tsc`. I have followed the readme instructions and generated files based on the same .proto

Full Logs:

```
❯ yarn tsc
yarn run v1.22.11
$ /Users/patriciahudakova/Projects/twirpscript_demo/node_modules/.bin/tsc
node_modules/twirpscript/dist/runtime/client/index.d.ts:73:28 - error TS2304: Cannot find name 'Headers'.

73     readonly headers: Pick<Headers, "get">;
                              ~~~~~~~

Found 1 error.

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

tsconfig:

```
{
    "compilerOptions": {
      "outDir": "./build/",
      "sourceRoot": "./src/",
      "rootDir": "./src/",
      "sourceMap": true,
      "inlineSources": true,
      "module": "commonjs",
      "moduleResolution": "node",
      "lib": [
        "es2019"
      ],
      "types": [
        "node"
      ],
      "target": "es2019",
      "experimentalDecorators": true,
      "esModuleInterop": true,
      "noUnusedParameters": true,
      "noImplicitReturns": true,
      "noFallthroughCasesInSwitch": true,
      "noUncheckedIndexedAccess": true,
      "noUnusedLocals": true,
      "forceConsistentCasingInFileNames": true
    },
    "include": [
      "./src/**/*.*"
    ],
    "exclude": [
      "node_modules"
    ],
    "strict": true,
    "compileOnSave": false
}
```

--- a/src/runtime/client/index.ts
+++ b/src/runtime/client/index.ts
@@ -90,10 +90,17 @@ interface RpcTransportOpts {
   body: string | Uint8Array | undefined | null;
 }
 
+/**
+ * Subset of [Headers](https://developer.mozilla.org/en-US/docs/Web/API/Headers) from Fetch API. Redefined so that end users don't need to include lib.dom.d.ts in server only applications.
+ */
+interface Headers {
+  get(name: string): string | null;
+}
+
 export interface RpcTransportResponse {
   arrayBuffer: () => Promise<ArrayBuffer>;
   json: () => Promise<unknown>;
-  readonly headers: Pick<Headers, "get">;
+  readonly headers: Headers;
   readonly ok: boolean;
   readonly status: number;
   text: () => Promise<string>;

